content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -176,7 +176,7 @@ The default value for `default-not-ready-toleration-seconds` and `default-unreac
 This admission controller will intercept all requests to exec a command in a pod if that pod has a privileged container.
 
 This functionality has been merged into [DenyEscalatingExec](#denyescalatingexec).
-The DenyExecOnPrivileged admission plugin is deprecated and will be removed in v1.18.
+The DenyExecOnPrivileged admission plugin is deprecated.
```

Comment:
This isn't _wrong_ but we should make the same fix for `DenyEscalatingExec` - I think I might have forgotten to make that clear.